docs/ballerina-by-example/examples/transform-json/transform-json.bal

```diff
@@ -36,8 +36,14 @@ function main (string[] args) {
     string city = "London";
 
     // Convert p of type Person to the response JSON, using the transformer defined earlier.
-    json<Person> response =? <json<Person>, updateCity(city)>p;
+    json<Person>|error response = <json<Person>, updateCity(city)>p;
```

Comment:
@omindu Transformer should be a safe conversion and error should not be returned but due to conversion issue now its returning an error. Ideally we should able to remove ? and just keep =